lumibot/data_sources/pandas_data.py

```diff
@@ -259,12 +240,13 @@ def _pull_source_symbol_bars(
         self,
         asset,
         length,
-        timestep=MIN_TIMESTEP,
+        timestep="",
```

Comment:
Data Type consistency. If the code treats everything like a string, putting None in as the default just adds additional complications.  The only reason for None to be so common in Lumibot generally is:
1. as defaults for Price (since $0.00 is possible in rare cases)
2. for input Objects where None is the only valid default.

Otherwise, consistent data typing is preferrable because the code will be safer and much less likely to throw NoneType errors.

Reply:
ok sounds like a good idea. maybe we should also be adding typing to functions now to take this a step further